deploy/services/README.md

```diff
@@ -20,6 +20,41 @@ The following services can be installed:
 * **certs** is used for storing the TLS certificates needed by the services.
 * **script** contains scripts for creating user accounts
 
+## Automated Certificate and Permission Setup
```

Comment:
Please move this into Installation Steps section and remove the manual commands.